ModelBundle/Repository/StatusRepository.php

```diff
@@ -217,4 +217,23 @@ protected function filterSearch(PaginateFinderConfiguration $configuration, Stag
 
         return $qa;
     }
+
+    /**
+     * @param array $statusIds
+     *
+     * @throws \Doctrine\ODM\MongoDB\MongoDBException
+     */
+    public function removeStatuses(array $statusIds)
+    {
+        $statusMongoIds = array();
+        foreach ($statusIds as $statusId) {
+            $statusMongoIds[] = new \MongoId($statusId);
+        }
```

Comment:
I prefer to use : 
array_walk($statusIds, function(&$item) {$item = new \MongoId($item);});
direct overriding $statusIds without temp variable and less verbose.